scassandra/src/main/scala-3/com/evolutiongaming/scassandra/LoadBalancingConfigImplicits.scala

```diff
@@ -0,0 +1,8 @@
+package com.evolutiongaming.scassandra
+
+import pureconfig.ConfigReader
+import pureconfig.generic.derivation.default.*
+
+trait LoadBalancingConfigImplicits {
+  implicit val configReaderLoadBalancingConfig: ConfigReader[LoadBalancingConfig] = ConfigReader.derived
```

Comment:
This fails at `apply from empty config` in `LoadBalancingConfigSpec` because [Scala 3 version of pureconfig doesn't support default values](https://github.com/pureconfig/pureconfig/issues/1488)...

Reply:
WIP here: https://github.com/pureconfig/pureconfig/pull/1599

Reply:
Tested it against above PR. It works fine and we don't need Scala version specific derivation code anymore. 

I think traits like `LoadBalancingConfigImplicits` make code tidier, so I'd keep them though.

Reply:
The mentioned PR is still stuck. I decided to remove pureconfig derivation.